app/i18n/cz/conf.php

```diff
@@ -183,7 +183,7 @@
 		'reading_view' => 'Switch to reading view',	// TODO - Translation
 		'rss_view' => 'Open RSS view in a new tab',	// TODO - Translation
 		'see_on_website' => 'Navštívit původní webovou stránku',
-		'shift_for_all_read' => '+ <kbd>Alt ⎇</kbd> to mark previous articles as read<br />+ <kbd>⇧ Shift</kbd> označí vše jako přečtené',	// TODO - Translation
```

Comment:
Is this a partial translation being reverted? I'm a bit confused.

Reply:
This is because of https://github.com/FreshRSS/FreshRSS/pull/2843 which added the first part of the current string. The problem is `make i18n-format` detects the strings is different than the English one and so, remove the `// TODO`